ignite/handlers/param_scheduler.py

```diff
@@ -226,6 +227,256 @@ def plot_values(cls, num_events: int, **scheduler_kwargs: Mapping) -> Any:
         return ax
 
 
+class StateParameterScheduler(ParamScheduler):
+    """An abstract class for updating a state parameter during
+    training not belonging to an Optimizer parameter group.
+
+    Args:
+        parameter_setter: function that sets the required parameter value.
+        param_name: name of parameter to update.
+        save_history: whether to log the parameter values to
+            `engine.state.param_history`, (default=False).
+
+    .. versionadded:: 0.6.0
+    """
+
+    def __init__(
+        self, parameter_setter: Callable, param_name: str, save_history: bool = False,
```

Comment:
I do not understand the role of this `parameter_setter`. If we work on `engine.state` then it is defined as 
```
parameter_setter = lambda engine, name, value: setattr(engine.state, name, value)
```

It is a callable but we do not have any idea of its signature and what it should return. Please fix that

Reply:
Thanks for your comment, I'll fix that